lib/botRepository.js

```diff
@@ -17,17 +18,25 @@ class ChatRepository {
 
   addChat(chatId) {
     this.redis.sadd(CHATS_SET_KEY, chatId);
-  };
+  }
 
   delChat(chatId) {
     this.redis.srem(CHATS_SET_KEY, chatId);
-  };
+  }
 
   getAllChats() {
     return Rx.Observable.fromPromise(this.redis.smembers(CHATS_SET_KEY))
       .flatMap(el => el);
   }
 
+  getLastPolledDate() {
```

Comment:
IDK, splitting in two repos would naturally lead to a super class that will simply handle the `redis` construction just to have a couple of methods...